code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -327,6 +324,289 @@ datum/reagent/toxin/staminatoxin/on_mob_life(mob/living/carbon/M)
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/venom/on_mob_life(var/mob/living/M as mob)
+	toxpwr = 0.05*volume
+	M.adjustBruteLoss((0.1*volume)*REM)
+	if(prob(15))
+		M.reagents.add_reagent("histamine",pick(5,10))
+		M.reagents.remove_reagent("venom",1)
+	..()
+
+datum/reagent/toxin/neurotoxin2
+	name = "Neurotoxin"
+	id = "neurotoxin2"
+	description = "Deals toxin and brain damage up to 60 before it slows down, causing confusion and a knockout after 54 elapsed cycles."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+datum/reagent/toxin/neurotoxin2/on_mob_life(var/mob/living/M as mob)
+	if(M.brainloss + M.toxloss <= 60)
+		M.adjustBrainLoss(1*REM)
+		M.adjustToxLoss(1*REM)
+	if(current_cycle == 54)
+		M.sleeping += 5
+	..()
+
+datum/reagent/toxin/cyanide
+	name = "Cyanide"
+	id = "cyanide"
+	description = "Deals toxin damage, alongside some oxygen loss. 8% chance of stun and some extra toxin damage."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.125 * REAGENTS_METABOLISM
+	toxpwr = 0.75
+
+datum/reagent/toxin/cyanide/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
+		M.losebreath += 1
+	if(prob(4))
+		M << "You feel horrendously weak!"
+		M.Stun(2)
+		M.adjustToxLoss(2*REM)
+	..()
+
+/datum/reagent/toxin/questionmark // food poisoning
+	name = "Bad Food"
+	id = "????"
+	description = "????"
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	toxpwr = 0.5
+
+datum/reagent/toxin/itching_powder
+	name = "Itching Powder"
+	id = "itching_powder"
+	description = "Lots of annoying random effects, chances to do some brute damage from scratching. 6% chance to decay into 1-3 units of histamine."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.4 * REAGENTS_METABOLISM
+	toxpwr = 0
+
+/datum/reagent/toxin/itching_powder/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
+	if(method == TOUCH)
+		M.reagents.add_reagent("itching_powder", volume)
+		return
+
+datum/reagent/toxin/itching_powder/on_mob_life(var/mob/living/M as mob)
+	if(prob(15))
+		M << "You scratch at your head."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(15))
+		M << "You scratch at your leg."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(15))
+		M << "You scratch at your arm."
+		M.adjustBruteLoss(0.2*REM)
+	if(prob(3))
+		M.reagents.add_reagent("histamine",rand(1,3))
+		M.reagents.remove_reagent("itching_powder",1)
+	..()
+
+datum/reagent/toxin/initropidril
+	name = "Initropidril"
+	id = "initropidril"
+	description = "Causes some toxin damage, 5% chances to cause stunning, suffocation, or immediate heart failure."
+	reagent_state = LIQUID
+	color = "#CF3600"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 2.5
+
+datum/reagent/toxin/initropidril/on_mob_life(var/mob/living/M as mob)
+	if(prob(5))
```

Comment:
Revert this if(prob(5)) shit.

Revert it on all the chems you added it to. It made all the toxins damn near worthless.

Initro is one of the ones you spend TCs on, why did you nerf it into the ground?